source/wp-content/themes/wporg-gutenberg/functions.php

```diff
@@ -319,6 +324,10 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		wp_enqueue_style( 'global-styles-css-custom-properties' );
 		wp_enqueue_style( 'wp-block-spacer' );
 
+		if ( wp_is_mobile() ) {
+			$block_editor_css = get_block_editor_theme_styles()[0]['css'];
+			wp_add_inline_style( 'wp-edit-post', $block_editor_css );
+		}
```

Comment:
It turns out this worked in the first place, except that it only worked on the sandbox,
locally the newly added styles would somehow be overridden.

Reply:
Does this have to be here? Can it be moved in the`if ( wp_is_mobile() )` statement above.

Reply:
My idea is to put the styles together, but it's also ok to move into the `if ( wp_is_mobile() )` statement above.
Done in [Fix several issues when loading page.](https://github.com/WordPress/wporg-gutenberg/pull/47/commits/8ff9f500274535186dfcf9353defd9763dba6e74)